sql/mozfun/norm/windows_version_info/udf.sql

```diff
@@ -0,0 +1,82 @@
+CREATE OR REPLACE FUNCTION
+  norm.windows_version_info( os STRING,
+    os_version STRING,
+    windows_build_number INT64 )
+  RETURNS STRING AS (
+    CASE
+      WHEN os NOT IN ('Windows_NT', 'Windows_95', 'Windows_98') THEN NULL
+      WHEN os = 'Windows_95' THEN 'Windows 95'
+      WHEN os = 'Windows_98' THEN 'Windows 98'
+      WHEN os_version = '4.0'
+    AND os = 'Windows_NT' THEN 'Windows NT 4.0'
```

Comment:
SQL formatter, what did you do 😭

Reply:
not sure, this is actually the google one because I cannot get ./bqetl format --check to run on my m1. I would very much like to take you up on your offer that once you're satisfied with the UDF itself that you could recommit with the formatter run.